### PR TITLE
Journalist can restrict premium articles for non-subscribers 

### PR DESCRIPTION
**Pull request by mauroavellaneda**
[PT](https://www.pivotaltracker.com/story/show/175248558)
## This PR contains:
- Feature test
- Checkbox in formulary

### CODE REVIEW

File: src/modules/articles.js [hunk 1]
```diff
@@ -1,19 +1,20 @@
 import axios from "axios";
 
 const Articles = {
-  async create(props) {
+  async create(title, lead, content, image, category, premium) {
     let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
-
+debugger
```
Comment (SaraFrantzen) [inline src/modules/articles.js]: remove dubugger

File: src/components/CreateArticlesForm.jsx [hunk 2]
```diff
@@ -13,61 +13,58 @@ const CreateArticlesForm = () => {
...
   const handleCategoryChange = (value) => {
-   setSelectedCategory(value);
-   return selectedCategory
+    setSelectedCategory(value);
   };
 
   const onSubmit = async (e) => {
     e.preventDefault();
-    let { title, lead, content, encodedImage } = e.target;
+    let { title, lead, content, encodedImage, premium} = e.target;
```
Comment (emtalen) [inline src/components/CreateArticlesForm.jsx]: are you getting encodedImage from the event target?

File: src/components/CreateArticlesForm.jsx [hunk 3]
```diff
@@ -13,61 +13,58 @@ const CreateArticlesForm = () => {
...
             label="Category"
             options={options}
-            onChange={(data) => {
+            onChange={(e, data) => {
               handleCategoryChange(data.value);
             }}
             data-cy="category"
           />
+          <Form.Field>
+            {" "}
```
Comment (emtalen) [inline src/components/CreateArticlesForm.jsx]: remove this please, I know it comes form the formatter

**Review by emtalen — CHANGES_REQUESTED** [review]

File: src/components/CreateArticlesForm.jsx (hunk 2 above)
Comment (mauroavellaneda) [reply src/components/CreateArticlesForm.jsx]: my bad, I should have declared outside.

**Review by emtalen — APPROVED** [review]

**Pull request merged**